pkg/error/httperror.go

```diff
@@ -56,6 +56,8 @@ func MakeErrorFromHTTP(resp *http.Response) error {
 		errCode = ErrorNotFound
 	case http.StatusConflict:
 		errCode = ErrorNameExists
+	case http.StatusRequestTimeout:
+		errCode = ErrorRequestTimedOut
```

Comment:
Is it intentional to use `TimedOut` instead of `Timeout`?

Reply:
Actualy yes, But I think Timeout will be a better option.